src/mapping/RadialBasisFctMapping.hpp

```diff
@@ -65,19 +81,29 @@ class RadialBasisFctMapping : public RadialBasisFctBaseMapping<RADIAL_BASIS_FUNC
 
   /// Treatment of the polynomial
   Polynomial _polynomial;
+
+  /// Ginkgo Configuration
+  MappingConfiguration::GinkgoParameter _ginkgoParameter;
```

Comment:
Please fix the Wreorder here.

Reply:
First time I see these errors, but will do!